docs/snippets/angular/button-group-story.ts.mdx

```diff
@@ -0,0 +1,32 @@
+```ts
+// ButtonGroup.stories.ts
+
+import { moduleMetadata } from '@storybook/angular';
+import { CommonModule } from '@angular/common';
+
+import ButtonGroup from './ButtonGroup.component';
+import Button from './button.component';
+import * as ButtonStories from './Button.stories';
+
+export default {
+  title: 'ButtonGroup',
+  component: ButtonGroup,
+  decorators: [
+    moduleMetadata({
+      declarations: [Button],
+      imports: [CommonModule],
+    }),
+  ],
+};
+
+const Template = (args: ButtonGroup) => ({
+  component:ButtonGroup,
```

Comment:
```suggestion
  component :ButtonGroup,
```

Do we need ensure we run prettier on these files?

Reply:
I'm going to check the snippets and see if any are left not properly indented. Still trying to wrap my head around why MDX and prettier aren't playing nice on my end.

Reply:
Yeah, maybe just something to figure out later, let's just get the snippets right for now.